data/json/itemgroups/locations.json

```diff
@@ -83,6 +83,13 @@
       ["material_niter", 20],
       ["material_limestone", 5]
     ]
+  },{
+    "type":"item_group",
+    "id":"prison_textile",
+    "subtype": "collection",
+    "entries" : [
+      { "item": "blanket", "prob": 25 },
+      { "item": "jumpsuit", "prob": 35 }
```

Comment:
I agree, open a PR or issue to track updating the prision definition to use new items if you want (I don't think those existed when the prison layout was added).